test/parking_location/configs/parking_location_choice_coeffs.csv

```diff
@@ -0,0 +1 @@
+coefficient_name,value,constrain
```

Comment:
The coefficient values in the spec file should be moved to the coefficients file.